domainobjs/ts/__tests__/commands.test.ts

```diff
@@ -1,120 +1,160 @@
 import { expect } from "chai";
 import { genRandomSalt } from "maci-crypto";
 
-import { Keypair, Message, PCommand, TCommand } from "..";
+import { PCommand, Keypair, TCommand } from "..";
 
-describe("Commands & Messages", () => {
+describe("Commands", () => {
   const { privKey, pubKey } = new Keypair();
   const k = new Keypair();
 
   const pubKey1 = k.pubKey;
-
   const newPubKey = k.pubKey;
```

Comment:
I see no difference between `pubKey1` and `newPubKey`?

Reply:
so a command takes a param `newPubKey"`(in case a user decides to change their MACI key) though in these tests we were not submitting any key change so totally right they are the same and can be simplified to keep only 1 key